src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,123 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleTitle = (event) => {
+    this.setState({
+      title: event.target.value,
+    });
+  }
```

Comment:
Можно попробовать вместо пяти идентичных методов написать один.
```suggestion
  handleChange = (event, name) => {
    this.setState({
      [name]: event.target.value,
    });
  }
```